media/windows/calendar-reservations/calendar-reservations.py

```diff
@@ -0,0 +1,217 @@
+        "id": 'c_bj96menjelb4pecracnninf45k@group.calendar.google.com',
+    },
+    {
+        "name" : "Test calendar #2",
+        "id" : 'c_ncm1ib261lp6c02i46mors4isc@group.calendar.google.com',
+    },
+    {
+        "name" : "Epiphany Events",
+        "id" : "churchofepiphany.com_9gueg54raienol399o0jtdgmpg@group.calendar.google.com",
+    }
+]
+
+#list of the domains the calendar will accept events from, will decline events from all others
+acceptable_domains = {
+    'cabral.org',
+    'epiphanycatholicchurch.org',
+    'churchofepiphany.com',
+}
+
+####################################################################
+#
+# Setup functions
+#
+####################################################################
+
+def setup_cli_args():
+    global default_app_json
+    tools.argparser.add_argument('--app-id',
+                                 default=default_app_json,
+                                 help='Filename containing Google application credentials')
+    global default_user_json
+    tools.argparser.add_argument('--user-credentials',
+                                 default=default_user_json,
+                                 help='Filename containing Google user credentials')
+
+    global verbose
+    tools.argparser.add_argument('--verbose',
+                                 action='store_true',
+                                 default=verbose,
+                                 help='If enabled, emit extra status messages during run')
+    global debug
+    tools.argparser.add_argument('--debug',
+                                 action='store_true',
+                                 default=debug,
+                                 help='If enabled, emit even more extra status messages during run')
+    global logfile
+    tools.argparser.add_argument('--logfile',
+                                 default=logfile,
+                                 help='Store verbose/debug logging to the specified file')
+
+    tools.argparser.add_argument('--donothing',
+                                 action='store_true',
+                                 help='....')
+           
+
+    global args
+    args = tools.argparser.parse_args()
+
+    # --debug implies --verbose
+    if args.debug:
+        args.verbose = True
+
+    return args
+
+####################################################################
+
+def check_response_status(events, calendar, log):
+    #iterate through the list of events and find ones where responseStatus = needsAction
+
+    needs_action_events = []
+    for event in events:
+        if "attendees" in event: #determines whether the event has any attendees
+            for attendee in event["attendees"]:
+                if attendee["email"] == calendar['id']:
+                    if attendee["responseStatus"] == "needsAction":
+                        needs_action_events.append(event)
+
+    log.info(f"Found {len(needs_action_events)} needsAction events from {calendar['name']} {calendar['id']}")
```

Comment:
```suggestion
    log.info(f"Found {len(needs_action_events)} needsAction events from {calendar['name']} (ID: {calendar['id']})")
```

Remember that this will be emitted for every page full of entries that are downloaded.  You might want to either adjust the verbiage or only emit at the end...?